contrib/tnitter/Makefile

```diff
@@ -1,3 +1,24 @@
-all:
+SERVER ?= localhost:8080
```

Comment:
why not using the real server on xymus? Especially if you want that apk automatically build for frdroid are fully functionnal